javascript/features/settings/setting_list.js

```diff
@@ -44,6 +44,7 @@ export const kSettingList = [
     /** Decoration related settings */
     new Setting('decorations', 'christmas_decorations', Setting.TYPE_BOOLEAN, false, 'Should The Strip be in a Christmas-y mood?'),
     new Setting('decorations', 'fantasy_island', Setting.TYPE_BOOLEAN, false, 'Is this the time for some fantasy?'),
+   new Setting('decorations', 'halloween_decorations', Setting.TYPE_BOOLEAN, false, 'Be afraid ... Be very afraid.'),
```

Comment:
Your tabbing seems a bit off here aswell.

Might be nice to note that the strip will be scary in the comment

Reply:
fixed